ee/packages/license/src/AirGappedRestriction.ts

```diff
@@ -0,0 +1,73 @@
+import EventEmitter from 'events';
+
+import { License } from '.';
+import { decryptStatsToken } from './token';
+
+const DAY_IN_MS = 24 * 60 * 60 * 1000;
+const NO_ACTION_PERIOD_IN_DAYS = 3;
+const WARNING_PERIOD_IN_DAYS = 7;
+
+class AirGappedRestrictionClass extends EventEmitter {
+	#restricted = true;
```

Comment:
nice

Reply:
This a review or a compliment? lol

Reply:
lol, just surprise, i think this is the first time i see someone using private fields on RC